app/src/main/java/com/globant/example/mentorapp/home/presentation/view/adapter/ListUsersAdapter.java

```diff
@@ -25,25 +26,25 @@
 
     private List<ModelUserEntity> users;
     private Context context;
-    private BaseRecyclerViewAdapter.onUserClick listener;
+    private ListUsersViewFragment fragmentListener;
```

Comment:
Keep a descriptive name like OnUserClickListener. I just pointed to move it to the fragment.